packages/web/src/lib/constants.ts

```diff
@@ -7,3 +7,11 @@ export const ALGORAND_SIGN_TRANSACTION_REQUEST = "algo_signTxn";
 export const mainnetURL = "https://node.algoexplorerapi.io";
 export const testnetURL = "https://node.testnet.algoexplorerapi.io";
 export const betanetURL = "https://node.betanet.algoexplorerapi.io";
+
+//official networks https://developer.algorand.org/docs/get-details/algorand-networks/
+export const mainnetGenesisHash = "wGHE2Pwdvd7S12BL5FaOP20EGYesN73ktiC1qzkkit8=";
+export const testnetGenesisHash = "SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=";
+export const betanetGenesisHash = "mFgazF+2uRS1tMiL9dsj01hJGySEmPN28B/TjjvpVW0=";
+
+//runtime network
+export const runtimeGenesisHash = "runtime";
```

Comment:
why not defining it as upper case?

Reply:
you mean the name of the const? `RUNTIME_GENESIS_HASH`?

Reply:
yes, but maybe the lower case is more readable.